bundles/org.openhab.binding.openthermgateway/README.md

```diff
@@ -0,0 +1,144 @@
+# OpenTherm Gateway binding
+
+This binding is used to integrate the OpenTherm Gateway into openHAB2.
```

Comment:
Leave out the 2 of openHAB please check for other occurrences on the README as well.

Reply:
Changed

Reply:
Changed